src/System.Net.Primitives/tests/FunctionalTests/System.Net.Primitives.Functional.Tests.csproj

```diff
@@ -42,6 +43,10 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
+    <!-- TODO: Remove this reference once packages are updated -->
```

Comment:
> Line 50 seems to point to a locally built pkgproj which, I expect contains the latest APIs

Maybe we're talking past each other.  When you add an entry to the project.json, you get the published package, not one built locally.  If you want one built locally, you need to add a P2P reference to the local package project, which is what L50 has, and which is what I added here that you commented on.

> Why is this reference to Requests needed for a Primitives contract test?

Because otherwise it ends up referencing the System.Net.Requests from the published package, which contains these same APIs, and the compiler complains that the same type is defined in two places.  Since I'm pulling in the live primitives build that has the types moved to it, I also need to pull in the live requests build that has the types moved from it.